docs/source/API/core/view/view.rst

```diff
@@ -344,6 +341,46 @@ Constructors
 
    Subview constructor. See ``subview`` function for arguments.
 
+.. cppkokkos:function:: explicit(traits::is_managed) View( const NATURAL_MDSPAN_TYPE& mds )
+
+   :param mds: the mdspan to convert from.
+
+   .. warning::
+
+      :cpp:`explicit(bool)` is only available on C++20 and later. When building Kokkos with C++17, this constructor will be fully implicit.
+      Be aware that later upgrading to C++20 will in some cases cause compilation issues in cases where :cpp:`traits::is_managed` is :cpp:`false`.
+
+   :cpp:`NATURAL_MDSPAN_TYPE` is the `natural mdspan <#Natural MDSpans>`_ of the View. The *natural mdspan* is only available if :cpp:type:`array_layout` is one of :cppkokkos:struct:`LayoutLeft`, :cppkokkos:struct:`LayoutRight`,
+   or :cpp:class:`LayoutStride`. This constructor is only available if *natural mdspan* is available.
+
+   Constructs a :cpp:class:`View` by converting from :cpp:any:`mds`. The :cpp:class:`View` will be unmanaged and constructed as if by :cpp:`View(mds.data(), array_layout_from_mapping(mds.mapping()))`
+
+   .. seealso:: `Natural MDSpans`_
+
+   .. versionadded:: 4.4.0
+
+.. cppkokkos:function:: template <class ElementType, class ExtentsType, class LayoutType, class AccessorType> explicit(SEE_BELOW) View(const mdspan<ElementType, ExtentsType, LayoutType, AccessorType>& mds)
+
+   :tparam ElementType: the MDSpan element type
+   :tparam ExtentsType: the MDSpan extents
+   :tparam LayoutType: the MDSpan layout
+   :tparam AccessorType: the MDSpan extents
```

Comment:
Why is it `MDSpan` instead of `mdspan`?

Reply:
Yeah I was trying to be more consistent with the title-case in one of the headings. I now made it consistently lowercase, and took the cowards way out and renamed the heading "MDSpan Conversion" to "Conversion to mdspan" to avoid the title case problem